awx/ui_next/src/api/models/Root.js

```diff
@@ -25,6 +25,14 @@ class Root extends Base {
   logout() {
     return this.http.get(`${this.baseUrl}logout/`);
   }
+
+  readAssetVariables() {
+    // TODO: There's better ways of doing this. Build tools, scripts,
```

Comment:
I'll fix this elsewhere after merging. (See related follow-up task issues in the PR summary)